packages/core/src/operator/operator-runtime/operator-v1/loadOperatorKeysFromGraph.ts

```diff
@@ -33,7 +33,8 @@ export const loadOperatorKeysFromGraph_V1 = async (
 
     // Load all operator addresses and pool addresses, pass in the whitelist to get them filtered
     // refereeConfig will be used to locally calculate the boos factor (hold tier thresholds and boostFactors as well as max staking capacity per key)
-    const { wallets, pools, refereeConfig } = await retry(() => getSentryWalletsForOperator(operator, operatorState.passedInOwnersAndPools));
+    const submissionQueryFilter = { eligibleForPayout: true, claimed: false, latestChallengeNumber: latestChallengeNumber };
```

Comment:
@craigbranscom We should not load submissions here in the `_V1` functions. Everything V1 will not know about bulkSubmissions.

Reply:
Replaced the submission filter with an empty object

Reply:
Looks good, thank you. Please go ahead and merge